ui/packages/ace/package.json

```diff
@@ -24,6 +24,7 @@
     "babel-preset-latest": "6.24.1",
     "babel-preset-react": "6.24.1",
     "babel-preset-stage-0": "6.24.1",
+    "chalk": "2.3.2",
```

Comment:
might be worth an nsp check on this version, if it is new.

Reply:
I didn't find any known vulnerabilities, other than hoek.

Reply:
cool thanks